composer.json

```diff
@@ -22,7 +26,6 @@
         "composer/composer": "^2.0"
     },
     "require-dev": {
-        "flarum/core": "dev-master",
```

Comment:
this is needed, in all bundled extension in fact

Reply:
In the monorepo, minimum stability dev + prefer stable + path repo is sufficient for installing from the local dev version.

Reply:
I see! neat